setup.py

```diff
@@ -64,4 +64,9 @@
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ),
+    project_urls={
+        'Source Code': 'https://github.com/certifi/python-certifi',
```

Comment:
"Source" is about three times as common as "Source Code": https://github.com/hugovk/pypi-tools/issues/11#issuecomment-549175199

```suggestion
        'Source': 'https://github.com/certifi/python-certifi',
```